### ISSUE TITLE
How  the `TransferTo` field of  kubernetes plugin used for?

### ISSUE THREAD

**Issue by DavadDi** [coredns/coredns#2286]
```
// Kubernetes implements a plugin that connects to a Kubernetes cluster.
type Kubernetes struct {
        // ...
	TransferTo         []string
}
```
```
func ParseStanza(c *caddy.Controller) (*Kubernetes, error) {
                //  .....
		case "transfer":
			tos, froms, err := parse.Transfer(c, false)
			if err != nil {
				return nil, err
			}
			if len(froms) != 0 {
				return nil, c.Errf("transfer from is not supported with this plugin")
			}
			k8s.TransferTo = tos
                        // ...
  ```

Only see  `TransferTo` is assigned on syntax parsed, but can't find any reference to using it? So what's the `TransferTo` used for? Any help?

**Comment by chrisohaver** [comment]
On the surface this looks like unfinished code.  There should be a check that validates the operation is allowed (e.g. as is done with `TransferAllowed` in file plugin ).  Currently, it appears that kubernetes plugin ignores the `transfer to` option is ignored, allowing transfers anywhere.

**Comment by chrisohaver** [comment]
fixed in #2292
